### PR TITLE
Mcm week2

### PR DESCRIPTION
**Pull request by mocathe1st**
Fairly basic but trying to follow the chapter 2 content.

### CODE REVIEW

File: src/main/java/com/bankofspring/model/Account.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+/**
+ * Class to represent a bank account
+ * @author malcolmmurray
+ *
+ */
+public class Account {
+	
+	private String accountNumber;
+	private String accountDescription;
+	private Double accountAmount; //This is a Double object for now as this will be needed later for Hibernate
```
Comment (seanclark84) [inline src/main/java/com/bankofspring/model/Account.java:14]: Just FYI, you shouldn't use a Double to represent currency in Java ([see here](http://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency))

File: src/main/java/com/bankofspring/model/AccountTransactions.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+/**
+ * Singleton class to record transactions
+ * @author Malcolm.Murray
+ *
+ */
+public class AccountTransactions {
+
+	private List<AccountTransaction> transactions;
+	
+	private static AccountTransactions instance;
```
Comment (seanclark84) [inline src/main/java/com/bankofspring/model/AccountTransactions.java:15]: Ah, I thought you had a single `AccountTransaction`, that would have been silly.

File: src/main/java/com/bankofspring/model/Account.java (hunk 1 above)
Comment (mocathe1st) [inline src/main/java/com/bankofspring/model/Account.java:14]: Ah, hadn't thought of that - good point though.

File: src/main/java/com/bankofspring/model/AccountTransactions.java (hunk 2 above)
Comment (mocathe1st) [inline src/main/java/com/bankofspring/model/AccountTransactions.java:15]: No, this is just a singleton container for all the transactions - ultimately these should be getting written to the DB instead.

**Pull request closed without merging**